osi_featuredata.proto

```diff
@@ -0,0 +1,260 @@
+syntax = "proto2";
+
+option optimize_for = SPEED;
+
+import "osi_common.proto";
+
+package osi;
+
+//
+// \brief Interface for sensor data, in contrast to interpreted data after object hypothesis and tracking.
+//
+// All information regarding the environment is given with respect to the sensor coordinate system specified in \c DetectionHeader::mounting_position. 
+// When simulating multiple sensors, each sensor has an individual copy of FeatureData in its own reference frame. This allows an independent treatment of the sensors.
+//
+message FeatureData
+{
+     // Lidar detection list for multiple lidar sensors.
```

Comment:
The indentation in this file is slightly garbled (switches between 5, 4 (the proper amount) or 3 spaces a number of times).